### DISCUSSION TITLE
WinUI 3 removing roadmap, referencing WinAppSDK's - will it update?

### DISCUSSION THREAD

**Discussion by mdtauk** [microsoft/WindowsAppSDK#3814] [Q&A] [answered]
https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md

This document still references 2022 and it's goals.

Microsoft is becoming less transparent and open with what it is working on, what it wants to add in the future, and letting the community play a role, whilst not driving these projects forward at a resource level that seems to enable vibrancy and innovation.

So will more effort be put into keeping the community informed and enthused going forward?

**Comment by chigy** [comment, marked as answer]
Thank you, @mdtauk for pointing this out. We acknowledge the issue and are in the process of updating this document and others like this.
